cartridge/eventloop-speedometer.lua

```diff
@@ -0,0 +1,64 @@
+local clock = require('clock')
+local fiber = require('fiber')
+
+local membership = require('membership')
+
+local vars = require('cartridge.vars').new('cartridge.eventloop-speedometer')
+
+vars:new('eventloop_speedometer', nil)
+vars:new('eventloop_speed_threshold', 0.5)
+vars:new('eventloop_speed_resolution', 0.2)
+vars:new('eventloop_speed_measure_every', 5)
+
+local function speedometer(resolution)
+    resolution = resolution or 0.2
+    local start = clock.monotonic64()
+    fiber.sleep(resolution)
+    local diff = clock.monotonic64() - start
+
+    local err = (diff - tonumber64(resolution*1e9))/1e9
+
+    return err
+end
+
+local function speedometer_fiber()
+    fiber.self():name("eventloop-speedometer")
+    while true do
+        membership.set_payload('slow', nil)
```

Comment:
In the worst case, this code will increment incarnation twice every 5 sec.

Reply:
Also, I don't like the idea of extending payload at all, see my considerations in https://github.com/tarantool/cartridge/issues/660